rust/sbp/src/messages/acquisition.rs

```diff
@@ -0,0 +1,397 @@
+// Copyright (C) 2015-2018 Swift Navigation Inc.
+// Contact: Swift Navigation <dev@swiftnav.com>
+//
+// This source is subject to the license found in the file 'LICENSE' which must
+// be be distributed together with this source. All other rights reserved.
+//
+// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+// EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+// WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+//****************************************************************************
+// Automatically generated from yaml/swiftnav/sbp/acquisition.yaml
+// with generate.py. Please do not hand edit!
+//****************************************************************************/
+// Satellite acquisition messages from the device.
+extern crate byteorder;
+#[allow(unused_imports)]
+use self::byteorder::{LittleEndian, ReadBytesExt};
+use super::gnss::*;
+
+// Satellite acquisition result
```

Comment:
Triple slash will cause the comments to be extracted from `crates.io` by `docs.rs` (as doc comments).

Reply:
Done